aDataGenerator/dataGenerator.js

```diff
@@ -0,0 +1,225 @@
+  "Myrtle Beach", "Daly City", "Port St. Lucie", "Albany", "Atlanta", "Newburgh", "Huntington", "Columbia",
+  "Moreno Valley", "Akron", "Fontana", "Riverside", "Grayslake", "Orlando", "Corpus Christi", "Lewisville",
+  "Fullerton", "Inglewood", "Honolulu", "Tampa", "Arlington", "Coral Springs", "Springfield", "Nashville",
+  "Monterey", "Hollywood", "Springfield", "Howell", "Elk Grove", "Lakeland", "Round Lake Beach", "Durham",
+  "Baltimore", "Saginaw", "Denton", "Bellevue", "Macon", "Shreveport", "Orlando", "Howell", "Cape Coral",
+  "Madison", "St. Paul", "Tyler", "Temecula", "Merced", "Tucson", "Virginia Beach", "Kenosha",
+  "Westminster", "Havre de Grace", "Apple Valley", "Fargo", "High Point", "Nashville", "Brighton",
+  "St. Paul", "Las Cruces", "Champaign", "Fayetteville", "Waterloo", "Santa Clara", "Macon", "Olympia",
+  "Knoxville", "Visalia", "Waterbury", "Beaumont", "Bremerton", "Port Orange", "Garland", "Bonita Springs",
+  "Providence", "Des Moines", "Omaha", "North Port", "Clarksville", "Norfolk",
+];
+
+const highlights = ['Sparkling clean', 'Great Location', 'Great check-in experience'];
+
+const optionalDescriptionSections = ['Guess access', 'Interaction with guests', 'Other things to note'];
+
+const amenitiesSections = ['Family features', 'Dining', 'Facilities', 'Guess access', 'Logistics', 'Bed and bath',
+  'Safety features', 'Outdoor'];
+
+const randomInterger = (min, max) => Math.floor(Math.random() * ((max - min) + 1)) + min;
+
+const arbitrarySentence = (arrEntry, numOfWords, period) => {
+  let outputString = '';
+  for (let i = 0; i < numOfWords; i += 1) {
+    if (i === 0) {
+      let word = arrEntry[randomInterger(0, arrEntry.length - 1)].split('');
+      word[0] = word[0].toUpperCase();
+      word = word.join('');
+      outputString += word;
+    } else {
+      outputString += ' ';
+      outputString += arrEntry[randomInterger(0, arrEntry.length - 1)];
+    }
+  }
+  if (period) {
+    outputString += '.';
+  }
+  return outputString;
+};
+
+const highlightsArray = () => {
+  const output = [];
+
+  highlights.forEach((title) => {
+    const obj = {
+      title,
+      comment: arbitrarySentence(loremIpsum, randomInterger(8, 16), true),
+    };
+    output.push(obj);
+  });
+  return output;
+};
+
+const descriptionArray = () => {
+  const output = [];
+  const spaceDescription = {
+    title: 'The space',
+    comment: arbitrarySentence(loremIpsum, randomInterger(192, 288), true),
+  };
+  output.push(spaceDescription);
+  optionalDescriptionSections.forEach((title) => {
+    const obj = {
+      title,
+      comment: arbitrarySentence(loremIpsum, randomInterger(86, 192), true),
+    };
+    output.push(obj);
+  });
+  return output;
+};
+
+const specialStayRangeArray = () => {
+  const output = [];
+  let count = 0;
+
+  for (let i = 0; i < randomInterger(0, 3); i += 1) {
+    const obj = {
+      startDate: new Date(2018, 7 + count, randomInterger(1, 15)),
```

Comment:
it'd be better practice to have the year be an input to the function so that it would work for years other than 2018